src/java.desktop/share/classes/javax/swing/plaf/synth/package-info.java

```diff
@@ -139,5 +139,7 @@
  *   <bind style="style" type="region" key="Table"/>
  *   <bind style="style" type="region" key="List"/>
  * }</pre>
+ *
+ * @since 1.5
```

Comment:
This is correct, I verified the `javax.swing.plaf.synth` package is available in [Java 5](https://docs.oracle.com/javase/1.5.0/docs/api/javax/swing/plaf/synth/package-summary.html) but it wasn't in Java 1.4.